doc/code/qml_qchem.rst

```diff
@@ -10,26 +10,10 @@ number observables.
 
 .. currentmodule:: pennylane.qchem
 
-.. rubric:: Modules
```

Comment:
I used this to list the modules instead of all the functions. But agree that having the functions listed here might by better.